doc/host-setup.md

```diff
@@ -28,6 +28,49 @@ You can set the image by passing `--set operator.satelliteSet.kernelModuleInject
 [InitContainer]: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
 [#137]: https://github.com/piraeusdatastore/piraeus-operator/issues/137
 
+### Build your own injector image
+
+If there is no injector image available for your distribution, you can instead build your own. The below example creates
+an injector image for Fedora CoreOS (FCOS).
+
+To build our injector image for FCOS extend the existing CentOS 8 injector image to automatically download and install
+the kernel headers. FCOS does not provide those headers and since FCOS lags behind the normal fedora releases,
+the matching package is not always available from the regular repos.
+
+This script will be the entrypoint to the image. It will download the kernel headers and invoke the DRBD build process:
```

Comment:
Mention that this should be called `entry-override.sh` here to make the steps easier to follow from top to bottom.